app/assets/stylesheets/govuk_publishing_components/components/_task-list.scss

```diff
@@ -106,17 +133,15 @@ $top-border: solid 2px $grey-3;
     border-bottom: solid $stroke-width $grey-2;
   }
 
-  .gem-c-task-list__step:last-child {
-    &:after {
-      // scss-lint:disable DuplicateProperty
-      height: -webkit-calc(100% - #{$gutter-half}); // fallback
-      height: calc(100% - #{$gutter-half});
-      // scss-lint:enable DuplicateProperty
-    }
+  &:after {
+    // scss-lint:disable DuplicateProperty
+    height: -webkit-calc(100% - #{$gutter-half}); // fallback
```

Comment:
Is this required? We use `calc` in other places without also including `-webkit-calc`?

Reply:
https://caniuse.com/#feat=calc doesn't look like we need it.

See also https://github.com/postcss/autoprefixer which uses caniuse data with a browser list to automatically figure this out for you.

Reply:
I should have put a comment in, will add. This is to correct a bug on iphone 4, which doesn't seem to obey calc properly. @nickcolley is there an alternative way of writing this out with autoprefixer? This was the only thing that would work.